nifi-standard-bundle/nifi-standard-processors/src/main/java/com/asymmetrik/nifi/processors/FetchFileSplits.java

```diff
@@ -0,0 +1,389 @@
+        .name("Include Header Line")
+        .description("Whether or not to read and include a header line that will be " +
+                "repeated for each split FlowFile")
+        .required(true)
+        .expressionLanguageSupported(true)
+        .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+        .allowableValues(new AllowableValue("true"), new AllowableValue("false"))
+        .defaultValue("false")
+        .build();
+    static final PropertyDescriptor LINES_PER_SPLIT = new PropertyDescriptor.Builder()
+        .name("Lines per Split")
+        .description("The number of lines to be included in each split, not including the " +
+                "header line (if applicable).")
+        .required(true)
+        .expressionLanguageSupported(true)
+        .addValidator(StandardValidators.INTEGER_VALIDATOR)
+        .defaultValue("1")
+        .build();
+    static final PropertyDescriptor FILENAME = new PropertyDescriptor.Builder()
+        .name("Filename")
+        .description("Name of the file that will be retrieved from the file system")
+        .required(true)
+        .expressionLanguageSupported(true)
+        .addValidator(StandardValidators.NON_BLANK_VALIDATOR)
+        .defaultValue("${filename}")
+        .build();
+    static final PropertyDescriptor COMPLETION_STRATEGY = new PropertyDescriptor.Builder()
+        .name("Completion Strategy")
+        .description("Specifies what to do with the original file on the file system once it has been pulled into NiFi")
+        .expressionLanguageSupported(false)
+        .allowableValues(COMPLETION_NONE, COMPLETION_MOVE, COMPLETION_DELETE)
+        .defaultValue(COMPLETION_NONE.getValue())
+        .required(true)
+        .build();
+    static final PropertyDescriptor MOVE_DESTINATION_DIR = new PropertyDescriptor.Builder()
+        .name("Move Destination Directory")
+        .description("The directory to the move the original file to once it has been fetched from the file system. This property is ignored unless the Completion Strategy is set to \"Move File\". "
+                + "If the directory does not exist, it will be created.")
+        .expressionLanguageSupported(true)
+        .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+        .required(false)
+        .build();
+    static final PropertyDescriptor CONFLICT_STRATEGY = new PropertyDescriptor.Builder()
+        .name("Move Conflict Strategy")
+        .description("If Completion Strategy is set to Move File and a file already exists in the destination directory with the same name, this property specifies "
+                + "how that naming conflict should be resolved")
+        .allowableValues(CONFLICT_RENAME, CONFLICT_REPLACE, CONFLICT_KEEP_INTACT, CONFLICT_FAIL)
+        .defaultValue(CONFLICT_RENAME.getValue())
+        .required(true)
+        .build();
+    static final PropertyDescriptor FILE_NOT_FOUND_LOG_LEVEL = new PropertyDescriptor.Builder()
```

Comment:
Is there value in specifying log level?  Generally, the log levels are tweaked in the logback.xml file.  In my opinion, routing to the relationships is enough and writing to the logs is superfluous.

Reply:
Agreed, but this was a hold-over from the current FetchFile processor, so I left it in with the same default values. That logic does seem better suited to logback, so I'll take these out for this processor.